developer_manual/digging_deeper/circles.rst

```diff
@@ -0,0 +1,96 @@
+=================
+Nextcloud Circles
+=================
+
+Circles is a Nextcloud App that allow Nextcloud users to generate their own grouping of users and contacts.
+
+The Circles app is installed by default since Nextcloud 22.
+It can be used by other apps for creating user managed groups called `circles`, an example of such an apps is the `Collectives` app.
+
+There is also the `Circles API reference <https://nextcloud.github.io/circles>`_ .
```

Comment:
I think it is at least the published parts (basically you use the `CirclesManager`). But I am not that into the internals of the circles app, at least that is how all apps I found use the circles API [(e.g. collectives).](https://github.com/nextcloud/collectives/blob/main/lib/Service/CircleHelper.php)

Reply:
Would agree that we should only add API documentation to docs.nextcloud.com that is actually covered by what we consider a public and stable API. Maybe we can just move this to the circles app for now?

In the end we should actually have this as an OCP API instead of requiring using OCA classes. 

@ArtificialOwl Iirc remember there was a ticket about that, but I can only find https://github.com/nextcloud/server/issues/31341#issuecomment-1150001181. I would be curious for what reason that was decided.

Reply:
I asked for an official API back then already when we had to implement it in Talk:
https://github.com/search?q=repo%3Anextcloud%2Fspreed%20OCA%5CCircles&type=code

So yeah, currently state of the art, but should not be in the official docs since it does not follow the 3 years deprecation cycle at the moment.